src/search/model/SearchUtils.ts

```diff
@@ -26,17 +26,23 @@ assertMainOrNode()
 
 const FIXED_FREE_SEARCH_DAYS = 28
 
+export const enum SearchCategoryTypes {
```

Comment:
we should probably put on our coding guidelines how we name enums (like single vs plural)